src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -737,31 +737,45 @@ public override ImmutableArray<CSharpAttributeData> GetAttributes()
                 var containingPEModuleSymbol = _containingType.ContainingPEModule;
 
                 // Could this possibly be an extension method?
-                bool alreadySet = _packedFlags.IsExtensionMethodIsPopulated;
-                bool checkForExtension = alreadySet
+                bool isExtensionAlreadySet = _packedFlags.IsExtensionMethodIsPopulated;
+                bool checkForExtension = isExtensionAlreadySet
                     ? _packedFlags.IsExtensionMethod
```

Comment:
One thing I really don't get about the original code here is why `checkForExtension` should be assigned `IsExtensionMethod` if the bit is already set. If the bit is already set, wouldn't you just say "don't bother checking, we've already populated it?"